beets/library.py

```diff
@@ -1467,8 +1467,10 @@ def tmpl_aunique(self, keys=None, disam=None):
 
         keys = keys or 'albumartist album'
         disam = disam or 'albumtype year label catalognum albumdisambig'
+        bracket = bracket or '[]'
         keys = keys.split()
         disam = disam.split()
+        bracket = None if bracket is ' ' else list(bracket)
```

Comment:
Right—I guess the question is whether the empty string might be a better "use nothing" indicator than a space.